auditbeat/module/file_integrity/fileinfo_posix.go

```diff
@@ -78,3 +84,19 @@ func NewMetadata(path string, info os.FileInfo) (*Metadata, error) {
 	}
 	return fileInfo, errs.Err()
 }
+
+func getExtendedAttributes(path string, dst map[string]*string) {
+	f, err := os.Open(path)
+	if err != nil {
+		return
+	}
+	defer f.Close()
+
+	for n, d := range dst {
+		att, err := xattr.FGet(f, n)
+		if err != nil {
+			continue
+		}
+		*d = string(att)
```

Comment:
While testing I noticed this can include the null-terminator. I think some trimming is needed.

> "selinux": "system_u:object_r:quota_db_t:s0\u0000"